### DISCUSSION TITLE
How to configure Parcel to use an SWC transformer-plugin?

### DISCUSSION THREAD

**Discussion by jhsware** [parcel-bundler/parcel#9236] [General] [answered]
Insanely fast React-like rendering-library [InfernoJS](https://infernojs.org/) has implemented an [SWC-plugin to transform](https://github.com/infernojs/swc-plugin-inferno) JSX-code to highly optimised Typescript.

How can I get Parcel to use it instead of the standard React-transformer?

The .swcrc file would look like this:

```
{
  "jsc": {
    "experimental": {
         "plugins": [
            ["swc-plugin-inferno", {
              // Options
            }]
        ],
    }
  }
}
```

**Comment by mischnic** [comment, marked as answer]
See https://github.com/parcel-bundler/parcel/issues/8613
